rpc/api/eth.go

```diff
@@ -78,6 +78,7 @@ var (
 		"eth_transact":                            (*ethApi).SendTransaction,
 		"eth_estimateGas":                         (*ethApi).EstimateGas,
 		"eth_call":                                (*ethApi).Call,
+		"eth_debugTransaction":                    (*ethApi).DebugTransaction,
```

Comment:
Please don't add anything to the `eth` RPC namespace. Debug stuff should go into `debug`.